src/index.ts

```diff
@@ -7,6 +7,22 @@ interface PopoverToggleTargetElementInvoker {
   popoverTargetElement: Element | null;
 }
 
+/** [MDN Reference](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/togglePopover **/
+type PopoverTogglePopoverOptionsBoolean = boolean;
```

Comment:
We do lose the helper that a boolean parameter is `force`, that was in the original signature `togglePopover(force: boolean)`. Is there a way to maintain that?

Reply:
Pushed in https://github.com/oddbird/popover-polyfill/pull/243/commits/33d148edb61395f1967f1655fe2d117494607f7a -- let me know if that works.